src/rules/resolutions.ts

```diff
@@ -15,9 +15,24 @@ import * as core from '@actions/core';
 import { PackageJson } from 'type-fest';
 
 export const validateResolutions = (packageJson: PackageJson) => {
-  if (packageJson.resolutions) {
+  const ignoredResolutions = core.getMultilineInput('ignore-resolutions');
+
+  const skipIgnoreResolutions =
+    !Array.isArray(ignoredResolutions) || ignoredResolutions.length === 0;
+  if (packageJson.resolutions && skipIgnoreResolutions) {
```

Comment:
`getMultilineInput()` already returns `string[]`
```suggestion
  const ignoredResolutions = core.getMultilineInput('ignore-resolutions');
  if (packageJson.resolutions && !ignoredResolutions.length) {
```

Reply:
Very true. I actually had the same thought, but didn't set up the default mock for the `@actions/core` method. 

Updated code and test to match the conditional👌